dbfit-java/core/src/main/java/dbfit/fixture/SetOption.java

```diff
@@ -0,0 +1,72 @@
+package dbfit.fixture;
```

Comment:
A general query about future direction: do you think we should now be creating new fixtures for all DbFit commands rather than using `SequenceFixture` methods or looking to use `SequenceFixture`'s 'System Under Test' (i.e. just wrap a `DBEnvironment` and expose its methods like `Fixture`s)?

If we create fixture for all commands then I think we'd no longer really need `DatabaseEnvironment` at all (standalone mode) and the only use of `SequenceFixture` in Flow Mode would be to auto rollback at the end of the page.